operations/enterprises.md

```diff
@@ -588,6 +588,39 @@ Adds a new space block to the specified space for a defined period of time.
 | --- | --- | --- | --- |
 | `SpaceBlockId` | string | required | Unique identifier of added [Space block](enterprises.md#space-block). |
 
+## Delete space blocks
+
+Removes specified space blocks from the spaces.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/spaceBlocks/delete`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "SpaceBlockIds": [
+    	"bf1e10b7-8a03-4675-9e27-05fc84312a58",
```

Comment:
incorrect format. (incorrect indentation)

Reply:
tab instead of spaces?